jira/cli/templates.go

```diff
@@ -52,12 +52,18 @@ comments:
 `
 const default_edit_template = `# issue: {{ .key }}
 update:
+{{if .overrides.comment}}
   comment:
-    - add: 
+    - add:
         body: |~
           {{ or .overrides.comment "" | indent 10 }}
+{{end}}
+{{ if .overrides.label }}
+  labels:
+    - add: {{ .overrides.label }}
+{{end}}
 fields:
-  summary: {{ or .overrides.summary .fields.summary }}
+  summary: "{{ or .overrides.summary .fields.summary }}"
```

Comment:
You are right.  The summary fields sometimes contain chars like ] or [  ie '[something]'  in the summary. 

This is another case where I could use a custom template, but I felt that as a total n00b coming in here, these things didn't quite work for me, and I may have just gone off frustrated to some other project that worked out of the box.  Thats why I thought the default templates should 'just work' for people who don't really understand how it works yet.  

The advanced user or someone who wants to dig into the source could very easily write up a template that works exactly as they want it to.  I had thought about adding a quoting function, whereby we could determine if there were meta characters in the string and escape them, but opted not to as this was quicker and seems to work for most cases.

Now that I think of it, it may be simple enough to just build a temporary yaml string and see if it parses as a string.  If not, then add the quotes, or make it a multi-line like we have above for comments.   So you'd have something like 

    summary: {{ or .overrides.summary .fields.summary | quote }}

And you'd get 

    summary: >-
             my [summary] that contains meta chars and possibly "quotes"

I did quick check in python and it parses there:

    >>> yaml.load("foo:\n  summary: >-\n    my [summary] that contains meta chars and possibly \"quotes\"")
    {'foo': {'summary': 'my [summary] that contains meta chars and possibly "quotes"'}}